system-tests/projects/plugin-before-browser-launch-deprecation/cypress/e2e/app.cy.js

```diff
@@ -8,5 +8,5 @@ it('asserts on browser args', () => {
     return
   }
 
-  cy.task('assertPsOutput')
+  cy.task('assertPsOutput', Cypress.browser.name)
```

Comment:
Why was this needed again?

Reply:
because we cannot pass extraneous arguments into the webdriver session or firefox will fail to start